scripts/env/dev/init.sh

```diff
@@ -52,14 +52,31 @@ check_supported_kernel() {
 
 #function to install/upgrade cortx-py-utils rpm
 install_cortx_py_utils() {
-  #rpm -q cortx-py-utils && yum remove cortx-py-utils -y && yum install cortx-py-utils -y
+  #install cpio
+  if rpm -q 'cpio' ; then
+    echo "cpio already present ... Skipping ..."
```

Comment:
make sure you have yum-utils - else install same as well

Reply:
fixed.